CharacterCreation/Class-Specific Documentation/Engineer/Engineer Repair Processes.txt

```diff
@@ -0,0 +1,463 @@
+once per Encounter. Regression Overrides do not stack.
+
+== System Restore   == 
+Potential Gain:35
+Action:     2 Actions
+Duration:   Permanent
+Range:      Touch
+
+    Can only be used on a cybernetic subject whose primary data center is still 
+intact that has "bled out" less than 8 Rounds ago. The Engineer makes a DC 60 
+Repair check. If successful, the subject is now "bleeding out". If the subject 
+is saved, they get the feat "Refurbished". Characters with the "Refurbished" 
+feat have a permanent -1 to Intelligence and can never again be revived with 
+System Restore.
+
+===== Material/Mechanic =====
+
+== Duct Tape    == 
+Potential Gain:40
+Action:     2 Actions
+Duration:   N/A
+Range:      Touch
+
+    You must have your Extruder drawn in order to use this ability. This process
+only works on a cybernetic patient. Your patient heals an ammount of health 
+listed on the table below. As your character learns more repair skill, the 
+better they heal. Unless otherwise stated, this ability cannot heal a patient 
+above half health.
+
++-----------------+--------------+--------+
+| Med Skill Above | Base Healing |  Dice  |
++-----------------+--------------+--------+
+|       9         |     20       | + 2d10 |
++-----------------+--------------+--------+
+|       19        |     30       | + 3d10 |
++-----------------+--------------+--------+
+|       29        |     35       | + 4d10 |
++-----------------+--------------+--------+
+|       39        |     40       | + 5d10 |
++-----------------+--------------+--------+
+|       49        |     50       | + 6d10 |
++-----------------+--------------+--------+
+
+== Weapon Tuning    == 
+Potential Gain:20
+Action:     1 Action
+Duration:   engagement
+Range:      Touch
+
+    You fiddle with your ally's gun quickly. You tune the whosits, and
+dial the whatsits. It now has a -1 to its Miss Chances in all range brackets and 
+has half the chance of jamming, rounded down. Normal use and wear on the gun will 
+almost definitely unalign what you've done by the end of the engagement, but 
+hopefully your efforts will help keep your ally alive until then.
+
+== Vehicle Repair   == 
+Potential Gain:40
+Action:     4 Actions
+Duration:   almost instant
+Range:      Touch
+
+    You pull out your tools and quickly repair the damage to a friendly
+vehicle. It gains 1 vehicle health. While this power is not required to repair
+vehicles, it does add a Potential reward for doing so. It also increases the 
+effect and may reduce the time taken to repair. 
+
+
+== Fortify  == 
+Potential Gain: 30
+Action:     2 Actions
+Duration:   N/A
+Range:      Touch
+
+    You set your extruder to emit a light, ballistic-absorbing crystalizing gel.
+It doesn't last very long in the elements and slowly turns to dust as the patient
+moves, but it reduces incoming kinetic and thermal engery wonderfully. It has
+carcenogenic proporties when used on organic matter.
```

Comment:
This will never ever be misused.

Reply:
I tried to make it hard to missuse effectively. Anything can be missused ineffectively, and sometimes that's what players want. (shrug)